src/fsharp/CompileOps.fs

```diff
@@ -2580,7 +2575,11 @@ type TcConfig private (data : TcConfigBuilder,validate:bool) =
             with _ -> 
                 error(Error(FSComp.SR.buildCannotReadAssembly(filename),rangeStartup))
         | _ ->
-            None, MSBuildResolver.HighestInstalledNetFrameworkVersionMajorMinor(), false
+            // TODO:  we have to get msbuild out of this
+            if data.useSimpleResolution then
```

Comment:
I think this whole thing needs to go before too long.  But I will add a #if for now.